gensim/models/ldamodel.py

```diff
@@ -301,17 +303,19 @@ def __init__(self, corpus=None, num_topics=100, id2word=None,
         self.minimum_phi_value = minimum_phi_value
         self.per_word_topics = per_word_topics
 
+        self.random_state = get_random_state(random_state)
```

Comment:
Oh, there was a reason, but it's not relevant any more. I'll just move it back to its old spot.